Makefile

```diff
@@ -14,22 +14,29 @@ SHAREPREFIX ?= ${PREFIX}/share/vis
 VERSION = $(shell git describe 2>/dev/null || echo "0.2")
 
 CONFIG_LUA ?= 1
+CONFIG_SREGEX ?= 0
 CONFIG_ACL ?= 0
 CONFIG_SELINUX ?= 0
 
+ifeq ($(CONFIG_SREGEX),1)
```

Comment:
The Makefile (as opposed to GNUmakefile) tries to only rely only on POSIX features, if possible this should be done in some other way.